docs/design-philosophy.md

```
@@ -378,6 +378,43 @@ GitHub API でしか設定できないため、`install.sh` から `gh api` で
+
+1. **配布先依存**: 言語・フレームワーク・ディレクトリ構造はプロジェクトごと。汎用設定は成立しない
+2. **責任分担**: vibecorp 配布物の品質は vibecorp 本体 CI で担保。配布先 CI に押し付けない
+3. **ユーザーの CI を尊重**: プロジェクト固有の CI 設計に vibecorp 専用ワークフローを混ぜない
+
+### 配布しないもの（統合問題対策として）
+
+- `.coderabbit.yaml` の統合観点拡張 — 配布先のディレクトリ構造依存
+- `.semgrep/rules/` — シェルスクリプト中心のプロダクトとミスマッチ、言語依存
+- shellcheck CI — `.claude/hooks/*.sh` のみ対象、vibecorp 本体 CI の責任範囲
+
+> 通常の PR レビューを支える `.coderabbit.yaml` は引き続き配布する（[CodeRabbit 設定（`.coderabbit.yaml`）](#coderabbit-設定-coderabbityaml) を参照）。本セクションが対象とするのは「統合問題を AI レビューで拾う目的の追加配布」であり、それは行わない。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)